pallets/staking/src/lib.rs

```diff
@@ -2656,6 +2656,27 @@ decl_module! {
                     .map(|p| p.intended_count = new_intended_count)
             })
         }
+
+
+        /// Declare no desire to either validate or nominate.
```

Comment:
This is the same description as the `chill` extrinsic.  The `chill` extrinsic is for the validator to call themselves, but this new one is for the governance council to force chill a validator.